src/main/java/com/cognifide/knotx/repository/RepositoryFacade.java

```diff
@@ -34,6 +34,7 @@
     @Override
     public void get(URI uri, AsyncResultHandler<Template<String, URI>> handler) throws IOException {
         repositoryConfiguration.getRepositories().stream()
+                .filter(metadata -> metadata.getType().validate(metadata))
```

Comment:
This way we will never know that user provided incorrect configuration. It will be just filtered out, without any trace in logs.